### PR TITLE
🎲 For #8505 - Update tab icon.

### PR DESCRIPTION
**Pull request by darkwing**
Single and double digit tabs:

Screenshot_20200224-211526
Screenshot_20200224-210815

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by ekager** [comment]
Out of scope of this update - but in theory we could probably start using the AC version of `TabCounter` now that our icon is more centered and doesn't need customization

File: app/src/main/res/drawable/ic_tabs.xml [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="24dp"
-    android:height="24dp"
-    android:viewportWidth="24"
-    android:viewportHeight="24">
-  <path
-      android:pathData="M15.22 22H5.78A3.78 3.78 0 0 1 2 18.22V8.78C2 6.69 3.7 5 5.78 5h9.44C17.31 5 19 6.7 19 8.78v9.44c0 2-1.7 3.78-3.78 3.78zM5.86 7C4.83 7 4 7.83 4 8.86v9.28C4 19.17 4.83 20 5.86 20h9.28c1 0 1.86-0.83 1.86-1.86V8.86C17 8 16.17 7 15 7H6zM6 4c0-1.1 1-2 2-2h8a6 6 0 0 1 6 6v0L22 16a2 2 0 0 1-2 2L20 8V8a4 4 0 0 0-4-4H6z"
-      android:fillColor="?primaryText" />
-</vector>
+android:width="24dp"
```
Comment (ekager) [inline app/src/main/res/drawable/ic_tabs.xml]: nit - this formatting looks a bit weird, can we format this?

Comment (darkwing) [reply app/src/main/res/drawable/ic_tabs.xml]: Updated formatting.

**Comment by darkwing** [comment]
@ekager Should we roll with this or rip out our stuff and go with AC's?

**Comment by ekager** [comment]
I think this is good to land for now! I'll file a followup

**Comment by ekager** [comment]
Opened #8791

**Pull request merged**